src/lib.rs

```diff
@@ -48,10 +76,38 @@ impl<T: Clone> MenuState<T> {
                 is_highlight: true,
             },
             events: Default::default(),
+            orientation: MenuOrientation::Right,
         }
     }
 
-    /// active the menu, this will select the first item
+    /// Sets the orientation for the menu.
+    ///
+    /// Default orientation is [MenuOrientation::Right].
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use tui_menu::{MenuState, MenuItem, MenuOrientation};
+    ///
+    /// let mut state = MenuState::<&'static str>::new(vec![
+    ///     MenuItem::item("Foo", "label_foo"),
+    ///     MenuItem::group("Group", vec![
+    ///         MenuItem::item("Bar 1", "label_bar_1"),
+    ///         MenuItem::item("Bar 2", "label_bar_1"),
+    ///     ])
+    /// ]).set_orientation(MenuOrientation::Left);
+    ///
+    ///
+    /// ```
+    ///
+    /// This is a fluent setter method which must be chained or used as it consumes self
+    #[must_use = "method moves the value of self and returns the modified value"]
+    pub const fn set_orientation(mut self, orientation: MenuOrientation) -> Self {
```

Comment:
For consitency with the methods on Menu, this should probably be just `orientation`, not set_orientation. This pattern is the same one we use in Ratatui, and is called the [builder lite pattern](https://matklad.github.io/2022/05/29/builder-lite.html).
```suggestion
    pub const fn orientation(mut self, orientation: MenuOrientation) -> Self {
```

If this is something that you need to change at runtime and not construction time, then it probably would be called `set_` and take `&mut self`.